admin/products.md

```diff
@@ -1,55 +1,58 @@
 # Products
-What you see, the customer sees. With Reaction, you can create, edit, and remove products inline, no separate admin panel required. It’s just simple, straight product management.
+What you see, the customer sees. Create, edit, and remove products inline— no separate admin panel required. It’s simple, straight product management.
 
 ## Creating Products
 To create a product, login as an admin, click on your profile in the top right corner, then click "Add Product" in the dropdown menu.
 
 ![](/assets/guide-products-add-product.png "Reaction Commerce create product")
+
 You’ll be taken to your product page, where you may click into and edit the available fields, as well as add variants.
 
 ![](/assets/guide-products-new-product.png "Reaction Commerce create product")
-You may also clone an existing product by clicking on the Clone icon (![](/assets/guide-icon-files.png "Reaction Commerce clone product")) from the grid view.
+
+You may also clone an existing product by clicking on the "Clone" icon (![](/assets/guide-icon-files.png "Reaction Commerce clone product")) from the grid view.
 
 ![](/assets/guide-products-cloneproduct.png "Reaction Commerce clone product")
+
 See sections below for details on editing products.
 
-Until you choose to make a product visible, it is hidden from the public by default. To make a product visible, click the “Make It Visible" link in the product detail view.
+By default, a product is hidden from public view until you choose to make it visible. To make a product visible, click on the “Make it visible" link in the product detail view.
 
 ![](/assets/guide-products-makevisible.png "Reaction Commerce make visible")
 
-Once the product is set to be visible, a link appears at the top of the product, which you may use to change back to hidden at any time.
+If you want to hide your product from public view, click on the "Make invisible" link at the top to change it back at any time.
 
 ![](/assets/guide-products-makeinvisible.png "Reaction Commerce make invisible")
 
 ## Editing Products
-There is no Reaction admin panel. Your product view is the same as the customer’s— just login, click on the product you wish to edit, and change any field.
+With Reaction Commerce, there aren't any admin panels. Your product view is the same as the customer’s, and it's all in real-time. Simply login, click on the product you wish to edit, and change any field.
 
```

Comment:
Is this still true? We moved a lot of stuff into a side panel which pops open when you edit, a lot of it is no longer inline.